### DISCUSSION TITLE
WASM support in the extension framework

### DISCUSSION THREAD

**Discussion by ossfellow** [cozodb/cozo#20] [Ideas]
Hello,
Thank you for the very interesting project, and a powerful data analysis tool.

To broaden accessibility, which would facilitate a richer extension ecosystem, it would be great if WebAssembly support could be added to the extension framework.
This will allow the user community to develop extensions in languages they’re already familiar with.

Thank you.

**Comment by zh217** [comment]
Great idea, especially considering the fact that the next release of the database itself runs on WASM (in addition to mobile platforms)!

We already have a working example -- see https://cozodb.github.io/wasm-demo/

**Reply by ossfellow** [reply]
Wonderful; so, you’ve already incorporated some state of the art features in very cool database.
Thank you for the great work 🍻
